src/interface_adapter/signup/SignupPresenter.java

```diff
@@ -0,0 +1,43 @@
+//
+// Source code recreated from a .class file by IntelliJ IDEA
+// (powered by FernFlower decompiler)
+//
+
+package interface_adapter.signup;
+
+import interface_adapter.ViewManagerModel;
+import interface_adapter.login.LoginState;
+import interface_adapter.login.LoginViewModel;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import use_case_signup.SignupOutputBoundary;
+import use_case_signup.SignupOutputData;
+
+public class SignupPresenter implements SignupOutputBoundary {
+    private final SignupViewModel signupViewModel;
+    private final LoginViewModel loginViewModel;
+    private ViewManagerModel viewManagerModel;
+
+    public SignupPresenter(ViewManagerModel viewManagerModel, SignupViewModel signupViewModel, LoginViewModel loginViewModel) {
+        this.viewManagerModel = viewManagerModel;
+        this.signupViewModel = signupViewModel;
+        this.loginViewModel = loginViewModel;
+    }
+
+    public void prepareSuccessView(SignupOutputData response) {
+        LocalDateTime responseTime = LocalDateTime.parse(response.getCreationTime());
+        response.setCreationTime(responseTime.format(DateTimeFormatter.ofPattern("hh:mm:ss")));
```

Comment:
I don't think we need to show creation time or response time, you could probably remove these two lines